client/src/components/UserContainer/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import './style.css';
+import HeaderWithMenu from '../HeaderWithMenu';
+
+class UserContainer extends Component {
+  state = {};
+
+  render() {
+    return <HeaderWithMenu />;
```

Comment:
@MohammedAlghazali Are you planning to add more to this component?

Reply:
yes, I will add other components